src/Symfony/Bundle/TwigBundle/Extension/ActionsExtension.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Bundle\TwigBundle\Extension;
 
+trigger_error('The '.__NAMESPACE__.'\ActionsExtension class is deprecated since version 2.2 and will be removed in Symfony 3.0.', E_USER_DEPRECATED);
```

Comment:
The solution might be to go the same way than for DialogHelper and TableHelper: trigger it in the constructor and add an optional constructor argument to disable it. This way, the bundle could disable the warning, but people register the extension manually would get the warning